tests/ops/qutrit/test_qutrit_channel_ops.py

```diff
@@ -113,28 +114,24 @@ def expected_jac_fn(p):
     @staticmethod
     def kraus_fn(p):
         """Gets a matrix of the Kraus matrices to be tested."""
-        return qml.math.stack(channel.QutritDepolarizingChannel(p, wires=0).kraus_matrices())
```

Comment:
I imported math from pennylane, I ended up removing all calls to `qml.math` and replaced with `math`